### PR TITLE
Add more information to Readme

### PR DESCRIPTION
**Pull request by Sharad24**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,3 +75,43 @@ trainer = Trainer(agent, env, mode="dyna", model="tabular", n_episodes=10000)
...
+
+## Algorithms
+
+### Deep RL
+ - DQN (Deep Q Networks)
+    - DQN
+    - Double DQN
+    - Dueling DQN
+    - Noisy DQN
+    - Categorical DQN
```
Comment (sampreet-arthi) [inline README.md:87]: Prioritized Replay DQN too

Comment (Sharad24) [reply README.md:87]: Can all of them work with it?

Comment (sampreet-arthi) [reply README.md:87]: Not yet. None of them can work with each other atm. There's an issue for this #209

Comment (Sharad24) [reply README.md:87]: So lets leave it then. Prioritized Replay is more of a utility. We might need to get rid of it completely after Reverb Intergration #233 #151

**Comment by Sharad24** [comment]
Going to merge this for now, lets keep getting in PRs.

**Pull request merged**